VFS4G builds failing in VS IDE on Windows

Sample error:

> error MSB4126: The specified solution configuration "|x64" is invalid. Please specify a valid solution configuration using the Configuration and Platform properties (e.g. MSBuild.exe Solution.sln /p:Configuration=Debug /p:Platform="Any CPU") or leave those properties blank to use the default solution configuration. [C:\github\VFSForGit\GVFS.sln]
1>C:\github\VFSForGit\GVFS\GVFS.Build\GVFS.PreBuild.csproj(185,5): error MSB3073: The command "call C:\github\VFSForGit\GVFS\GVFS.Build\..\..\Scripts\RestorePackages.bat" exited with code 1.

@nickgra I'm also getting an issue running the unit tests in VS on Windows.  Do you see the same?

```
10>C:\Users\jeschu\gvfs3\BuildOutput\GVFS.Platform.Mac\obj\x64\Debug\netcoreapp2.1\GVFS.Platform.Mac.AssemblyInfo.cs(14,12,14,54): error CS0579: Duplicate 'System.Reflection.AssemblyCompanyAttribute' attribute
10>C:\Users\jeschu\gvfs3\BuildOutput\GVFS.Platform.Mac\obj\x64\Debug\netcoreapp2.1\GVFS.Platform.Mac.AssemblyInfo.cs(15,12,15,60): error CS0579: Duplicate 'System.Reflection.AssemblyConfigurationAttribute' attribute
10>C:\Users\jeschu\gvfs3\BuildOutput\GVFS.Platform.Mac\obj\x64\Debug\netcoreapp2.1\GVFS.Platform.Mac.AssemblyInfo.cs(18,12,18,54): error CS0579: Duplicate 'System.Reflection.AssemblyProductAttribute' attribute
10>C:\Users\jeschu\gvfs3\BuildOutput\GVFS.Platform.Mac\obj\x64\Debug\netcoreapp2.1\GVFS.Platform.Mac.AssemblyInfo.cs(19,12,19,52): error CS0579: Duplicate 'System.Reflection.AssemblyTitleAttribute' attribute
```

> 10>C:\Users\jeschu\gvfs3\BuildOutput\GVFS.Platform.Mac\obj\x64\Debug\netcoreapp2.1\GVFS.Platform.Mac.AssemblyInfo.cs(14,12,14,54): error CS0579: Duplicate 'System.Reflection.AssemblyCompanyAttribute' attribute

@derrickstolee and I were both able to fix by deleting the BuildOutput dir

--- a/GVFS.sln
+++ b/GVFS.sln
@@ -180,6 +180,9 @@ EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GVFS.FunctionalTests.LockHolder", "GVFS\GVFS.FunctionalTests.LockHolder\GVFS.FunctionalTests.LockHolder.csproj", "{FA273F69-5762-43D8-AEA1-B4F08090D624}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GVFS.Hooks.Mac", "GVFS\GVFS.Hooks\GVFS.Hooks.Mac.csproj", "{4CC2A90D-D240-4382-B4BF-5E175515E492}"
+	ProjectSection(ProjectDependencies) = postProject
+		{A4984251-840E-4622-AD0C-66DFCE2B2574} = {A4984251-840E-4622-AD0C-66DFCE2B2574}
+	EndProjectSection
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GVFS.Upgrader", "GVFS\GVFS.Upgrader\GVFS.Upgrader.csproj", "{AECEC217-2499-403D-B0BB-2962B9BE5970}"
 EndProject

--- a/GVFS/GVFS.Build/GVFS.PreBuild.csproj
+++ b/GVFS/GVFS.Build/GVFS.PreBuild.csproj
@@ -182,7 +182,7 @@
           Outputs="$(RestoreTimestampPath)"
           DependsOnTargets="GVFSPreBuild"
           Condition="'$(OS)' == 'Windows_NT'">
-    <Exec Command="call $(MSBuildThisFileDirectory)..\..\Scripts\RestorePackages.bat"
+    <Exec Command="call $(MSBuildThisFileDirectory)..\..\Scripts\RestorePackages.bat $(Configuration)"
           WorkingDirectory="$(MSBuildThisFileDirectory)../../" />
     <Touch Files="$(RestoreTimestampPath)" AlwaysCreate="true" />
   </Target>

--- a/Scripts/RestorePackages.bat
+++ b/Scripts/RestorePackages.bat
@@ -3,6 +3,10 @@ CALL %~dp0\InitializeEnvironment.bat || EXIT /b 10
 
 SETLOCAL
 
+IF "%1"=="" (SET "Configuration=Debug") ELSE (SET "Configuration=%1")
+ 
+SET SolutionConfiguration=%Configuration%.Windows
+
 SET nuget="%VFS_TOOLSDIR%\nuget.exe"
 IF NOT EXIST %nuget% (
   mkdir %nuget%\..

